.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Feature Request
+description: Suggest an idea to enhance the current functionality of PyCytominer.
+title: "FeatRequest: "
```

Comment:
Is this something that is preset? Why not just spell out `Feature`?

Reply:
I thought it would be too long when I first made this, and the Pandas templates call it `FR` and I didn't want to use that. I think now it isn't too long after a few days after creating this, I will make it `FeatureRequest`.